bin/test-workflow/0_prep_env.sh

```diff
@@ -4,14 +4,14 @@ set -o pipefail
 
 ### PLATFORM DETAILS
 export CONJUR_OSS_HELM_INSTALLED="${CONJUR_OSS_HELM_INSTALLED:-true}"
+export UNIQUE_TEST_ID="$(uuidgen | tr "[:upper:]" "[:lower:]" | head -c 10)"
```

Comment:
Ha! I was just writing a comment that this was missing! You beat me to the punch. :)